index.html

```diff
@@ -158,50 +207,65 @@ <h2 class="lets-connect con" >LET'S CONNECT</h2>
    <li>   <img src="assets//images/medium.svg" alt="medium"></li>
 <li>   <img src="assets//images/github.svg" alt="github"> </li>
 <li>    <img src="assets//images/angellist.svg" alt="angellist"></li>
-</ul><br><br>   
-<button type="button" class="project1-btn get-btn">Get My resume</button>
-<br><br>
+</ul></div><br><br>   
+<button type="button" class="getmyre-btn get-btn">Get My resume</button>
+<br><br> </div>
+<div class="aboutsf">
+    <div class="lang-arrow">
 <h1 class="mv3">Languges</h1>
-<img class="arrow" src="./assets/images/arrow.svg" alt="arrow">
-<ul class="language1">
-    <li class="lang1">
+<img class="arrow" src="./assets/images/arrow.svg" alt="arrow"> </div>
+<ul class="language1 group-f">
+    <li class="lang1"> 
+        <img class="ellipse connect-via social-mv" src="./assets/images/Ellipse_1_js.svg" alt="js">
```

Comment:
- [ ] Please try to match the design of this section to the one provided in Figma.  I recommend to decreasing the boldness of headers in this section and justifying all the content to the left: 

Your design  | Figma design
------------- | -------------
<img width="580" alt="Screenshot 2022-02-17 at 17 18 14" src="https://user-images.githubusercontent.com/77038610/154489705-64957498-378e-4b36-82e3-78ab023ea24a.png"> | <img width="548" alt="Screenshot 2022-02-17 at 17 19 00" src="https://user-images.githubusercontent.com/77038610/154489812-801fd111-56f3-4c84-ab4f-a6cd55674e97.png">

- [ ] Please add the underline borders below `Languages, Frameworks, Skills` as provided in figma and try to have similar space between the headings. 

- [ ] In this section there are 2 blocks  and the left one should take 40, the right one should take remaining 60% of the page's width. In your case they are taking equal space.please make the two sections take 40%/ 60% space. 
- [ ] It is recommend to use `<h1>` tag only once in a page, therefore, I recommend you wrapping the content with other header tags like `<h2>, <h3>`